backend/lib/endpoints/posts.js

```diff
@@ -29,6 +29,9 @@ async function routes(app) {
   const UNLOGGED_POST_SIZE = 120;
   const EXPIRATION_OPTIONS = ["day", "week", "month"];
 
+  // @desc    Get all posts
```

Comment:
not sure why you added this now, seems out of scope of the pull request and it might cause more merge conflicts with other PR's 
not saying this is bad but maybe it's better to open a separate issue for this and look at the possibilities to improve code documentation instead of just adding it

Reply:
and if we decide to use this we could add everywhere in the API
also maybe we should chedk the integration with the `fastify-oas` plugin because this can help improve maybe also the API documentation that is automatically generated and can be seen with the Swagger UI?

Reply:
Sorry. I already created a branch for the documentation. Didn't know it'll cause issues here.